### DISCUSSION TITLE
error while importing geocube.rasterize

### DISCUSSION THREAD

**Discussion by emaildipen** [corteva/geocube#85] [Q&A]
ImportError                               Traceback (most recent call last)
<ipython-input-1-b32d5f425c1e> in <module>
      6 
      7 from geocube.api.core import make_geocube
----> 8 from geocube.rasterize import rasterize_points_griddata, rasterize_points_radia

ImportError: cannot import name 'rasterize_points_radia' from 'geocube.rasterize' (C:\ProgramData\Anaconda3\envs\pratice\lib\site-packages\geocube\rasterize.py)

**Comment by snowman2** [comment]
Please provide this information:

#### Installation method/steps
 - Installation method (conda, pip, from source, etc...)
 - Please provide all commands/steps you used to install geocube and GDAL/rasterio/fiona.

#### Environment Information
 - rioxarray version you are attempting to install
 - rasterio version (`rio --version`)
 - rasterio GDAL version (`rio --gdal-version`)
 - fiona version (`fio --version`)
 - fiona GDAL version (`fio --gdal-version`)
 - Python version (`python -c "import sys; print(sys.version.replace('\n', ' '))"`)
 - Operation System Information (`python -c "import platform; print(platform.platform())"`)

**Comment by emaildipen** [comment]
conda install geocube 

rioxarray (0.9.0) deps:
  rasterio: 1.1.0
    xarray: 0.19.0
      GDAL: 3.0.2

Other python deps:
     scipy: 1.3.1
    pyproj: 2.6.1.post1

**Reply by snowman2** [reply]
It might be worthwhile to try creating a new conda environment and starting fresh. Pyproj has useful tips: https://pyproj4.github.io/pyproj/stable/installation.html
